app/models/social_messaging/twitter_status.rb

```diff
@@ -32,13 +32,9 @@ def category
       post.channel.twitter_hashtag
     end
 
-    def host
-      ENV.fetch('HOST')
-    end
-
     def status
       "#{title} #{Rails.application.routes.url_helpers.post_url(titled_slug: post.to_param,
-                                                                host: host)} via @#{name} #til ##{category}"
+                                                                host: 'til.magmalabs.io')} via @#{name} #til ##{category}"
```

Comment:
then why we need `ENV['HOST']`? if we hardcode til.magmalabs.io :p

Reply:
If you see closely, I removed that method ;)